### DISCUSSION TITLE
Will there be further development for gnome < 40?

### DISCUSSION THREAD

**Discussion by TentativeConvert** [material-shell/material-shell#684] [Q&A] [answered]
If I understand correctly, the lastest version is only compatible with gnome >= 40.  Will there be any further material releases for older versions of gnome?  (I am running material 12 on gnome 3.36.)

**Comment by PapyElGringo** [comment, marked as answer]
Since Material Shell alter gnome shell it's very complicated to support several different version. If the changes are small between release we can add conditionnal fixes depending on which gnome shell version the user use but when big changes occur it's not possible. Therefore new devs are only applied to last gnome shell versions.
But you should see that Material Shell upgrades will come as the same time of the gnome shell ones.

**Reply by TentativeConvert** [reply]
I was mainly hoping for bug fixes, not any additional features.  The features in version 12 are already awesome :) It's definitely the best desktop I've ever had, just a bit unstable.  Thanks a lot for building this, and for making it available! It feels like the future has just arrived.
